mahta/CSS/CSS responsive design/styles.css

```diff
@@ -0,0 +1,40 @@
+ @media only screen and (max-width: 600px) and (orientation: landscape) {
+   body{
+      background-color: lightgoldenrodyellow;
```

Comment:
I already wondered how do you find the colors you use. Now I also wonder how long a color name can be in CSS.

Reply:
Vs code's recomendations.. but I think they'll get me bored soon and I'll go back to rgbs😁

Reply:
But there is also this site I've been checking out, it has way longer names for colors. Trust me😂